vm-virtio/src/console.rs

```diff
@@ -365,6 +466,7 @@ impl VirtioDevice for Console {
             };
         self.kill_evt = Some(self_kill_evt);
 
+        let cb = interrupt_cb.clone();
```

Comment:
No need to clone the interrupt callback since it should be called once from here. There is no need for the callback to be called from inside the new thread.

Reply:
Agreed.